### PR TITLE
ISSD-2730 [PRM] Create a custom fragment to map the user associated with MDF Claim

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [ISSD-2730](https://liferay.atlassian.net/browse/ISSD-2730)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json [hunk 1]
```diff
@@ -897,6 +883,108 @@
...
+																				"en_US": "<p>Partner Information</p>\n",
+																				"es_ES": "Invoice",
+																				"fi_FI": "Invoice",
+																				"fr_FR": "Invoice",
+																				"hu_HU": "Invoice",
+																				"ja_JP": "Invoice",
+																				"nl_NL": "Invoice",
+																				"pt_BR": "Invoice",
+																				"sv_SE": "Invoice",
+																				"zh_CN": "Invoice"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json:938]:
```suggestion
																				"ar_SA": "<p>Partner Information</p>\n",
																				"ca_ES": "<p>Partner Information</p>\n",
																				"de_DE": "<p>Partner Information</p>\n",
																				"en_US": "<p>Partner Information</p>\n",
																				"es_ES": "<p>Partner Information</p>\n",
																				"fi_FI": "<p>Partner Information</p>\n",
																				"fr_FR": "<p>Partner Information</p>\n",
																				"hu_HU": "<p>Partner Information</p>\n",
																				"ja_JP": "<p>Partner Information</p>\n",
																				"nl_NL": "<p>Partner Information</p>\n",
																				"pt_BR": "<p>Partner Information</p>\n",
																				"sv_SE": "<p>Partner Information</p>\n",
																				"zh_CN": "<p>Partner Information</p>\n"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/mdf-claim-info-contact/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
+[#assign mdfClaimId = (request.getAttribute("INFO_ITEM").objectEntryId)! /]
+
+[#if mdfClaimId?has_content]
+	[#assign mdfClaim= (restClient.get("/c/mdfclaims/${mdfClaimId}?nestedFields=user"))! /]
+
+[#assign partnerAccount = (mdfClaim.r_usrToMDFClms_user.accountBriefs[0].name)! /]
+	[#assign firstName = (mdfClaim.r_usrToMDFClms_user.givenName)! /]
+	[#assign lastName = (mdfClaim.r_usrToMDFClms_user.familyName)! /]
+	[#assign infoEmail = (mdfClaim.r_usrToMDFClms_user.emailAddress)! /]
+[#assign phone = (mdfRequest.r_usrToMDFReqs_user.userAccountContactInformation.telephones?map(telephone -> telephone.phoneNumber)?join(', '))! /]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/mdf-claim-info-contact/index.html:10]:
```suggestion
    [#assign partnerAccount = (mdfClaim.r_usrToMDFClms_user.accountBriefs[0].name)! /]
	[#assign firstName = (mdfClaim.r_usrToMDFClms_user.givenName)! /]
	[#assign lastName = (mdfClaim.r_usrToMDFClms_user.familyName)! /]
	[#assign infoEmail = (mdfClaim.r_usrToMDFClms_user.emailAddress)! /]
    [#assign phone = (mdfRequest.r_usrToMDFReqs_user.userAccountContactInformation.telephones?map(telephone -> telephone.phoneNumber)?join(', '))! /]
```

Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/mdf-claim-info-contact/index.html:10]:
```suggestion
    [#assign partnerAccount = (mdfClaim.r_usrToMDFClms_user.accountBriefs[0].name)! /]
	[#assign firstName = (mdfClaim.r_usrToMDFClms_user.givenName)! /]
	[#assign lastName = (mdfClaim.r_usrToMDFClms_user.familyName)! /]
	[#assign infoEmail = (mdfClaim.r_usrToMDFClms_user.emailAddress)! /]
    [#assign phone = (mdfRequest.r_usrToMDFReqs_user.userAccountContactInformation.telephones?map(telephone -> telephone.phoneNumber)?join(', '))! /]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json [hunk 3]
```diff
@@ -792,24 +783,19 @@
...
-														"widthType": "Fluid"
+														"paddingTop": 2
 													}
 												},
 												"pageElements": [
 													{
 														"definition": {
 															"fragment": {
-																"key": "DIALECT-dialect-text"
+																"key": "dialect-dialect-text"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json:798]:
```suggestion
																"key": "DIALECT-dialect-text"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json [hunk 4]
```diff
@@ -897,6 +883,108 @@
...
+														"paddingLeft": 1,
+														"paddingRight": 2,
+														"paddingTop": 2
+													}
+												},
+												"pageElements": [
+													{
+														"definition": {
+															"fragment": {
+																"key": "dialect-dialect-text"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json:913]:
```suggestion
																"key": "DIALECT-dialect-text"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json (hunk 1 above)
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json:938]:
```suggestion
																				"ar_SA": "<p>Partner Information</p>\n",
																				"ca_ES": "<p>Partner Information</p>\n",
																				"de_DE": "<p>Partner Information</p>\n",
																				"en_US": "<p>Partner Information</p>\n",
																				"es_ES": "<p>Partner Information</p>\n",
																				"fi_FI": "<p>Partner Information</p>\n",
																				"fr_FR": "<p>Partner Information</p>\n",
																				"hu_HU": "<p>Partner Information</p>\n",
																				"ja_JP": "<p>Partner Information</p>\n",
																				"nl_NL": "<p>Partner Information</p>\n",
																				"pt_BR": "<p>Partner Information</p>\n",
																				"sv_SE": "<p>Partner Information</p>\n",
																				"zh_CN": "<p>Partner Information</p>\n"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json [hunk 5]
```diff
@@ -939,23 +1027,17 @@
...
-														"widthType": "Fluid"
+														"paddingTop": 2
 													}
 												},
 												"pageElements": [
 													{
 														"definition": {
 															"fragment": {
-																"key": "DIALECT-dialect-text"
+																"key": "dialect-dialect-text"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/mdf-claim-details-template/page-definition.json:1040]:
```suggestion
																"key": "DIALECT-dialect-text"
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by jairmedeiros — CHANGES_REQUESTED** [review]
small changes

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:forward:force

**Pull request closed without merging**